Apps/frontend/src/main.js

```diff
@@ -1,9 +1,14 @@
 import "./app.scss";
 // @ts-ignore
 import App from "./App.svelte";
+import { channelConfig } from "./stores";
```

Comment:
Unused import?
Try going into `Apps/frontend` and executing `npm run prettier:fix` and `npm run ling:fix`. The latter should show you unused imports and such.